src/main/java/com/pinterest/secor/tools/ProgressMonitor.java

```diff
@@ -207,8 +207,14 @@ private void exportToStatsD(List<Stat> stats) {
                     assert committedOffset <= lastOffset: Long.toString(committedOffset) + " <= " +
                         lastOffset;
 
-                    long offsetLag = lastOffset - committedOffset;
-                    long timestampMillisLag = lastTimestampMillis - committedTimestampMillis;
+                    long offsetLag = 0L;
```

Comment:
Hmm...perhaps not return, but it could `continue` to skip this `TopicPartition`?